conjur.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
 
 
   gem.add_dependency 'activesupport'
-  gem.add_dependency 'conjur-api', '~> 4.20', '>= 4.20.1'
+  gem.add_dependency 'conjur-api', '>= 4.20.3'
```

Comment:
I wasn't sure how multiple version specifiers work.  Does it just check that they match in order?